apps/app_dial.c

```diff
@@ -88,9 +88,12 @@
 				</argument>
 				<xi:include xpointer="xpointer(/docs/info[@name='Dial_Resource'])" />
 			</parameter>
-			<parameter name="timeout" required="false">
+			<parameter name="timeout" required="false" argsep="|">
```

Comment:
As was mentioned in a comment, | is discouraged - why the use of it here? Is there an alternative?

Reply:
That was somewhat arbitrary, I've changed it to `^`, which I think is more common as the secondary arg delimiter.